storage/storage.cpp

```diff
@@ -770,6 +771,11 @@ void Storage::RegisterCountryFiles(TIndex const & index, string const & director
   TLocalFilePtr localFile = GetLocalFile(index, version);
   if (localFile)
     return;
+  // @TODO(bykoianko) This place should be implemented faster.
+  // You use Find twice. Here and in GetCountryFile.
```

Comment:
Ah, sorry, I just saw that it's a RegisterCountryFiles() method. In any case, second call to Find is not a bottleneck.